main/options/options.cc

```diff
@@ -27,6 +27,7 @@ struct PrintOptions {
 const vector<PrintOptions> print_options({
     {"parse-tree", &Printers::ParseTree},
     {"parse-tree-json", &Printers::ParseTreeJson},
+    {"parse-tree-ruby", &Printers::ParseTreeRuby},
```

Comment:
do other tools other than whitequark use this format?
If not, WDYT about `parse-tree-whitequark-like` The `like` there is for "best effort"